fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBiomeTags.java

```diff
@@ -0,0 +1,155 @@
+	 */
+	public static final TagKey<Biome> IS_PLAINS = register("is_plains");
+	public static final TagKey<Biome> IS_SAVANNA = register("is_savanna");
+	/**
+	 * For land biomes where ice naturally spawns.
+	 * For biomes where snow alone spawns, see {@link ConventionalBiomeTags#IS_SNOWY}.
+	 */
+	public static final TagKey<Biome> IS_ICY = register("is_icy");
+	/**
+	 * For water biomes where ice naturally spawns.
+	 * For biomes where snow alone spawns, see {@link ConventionalBiomeTags#IS_SNOWY}.
+	 */
+	public static final TagKey<Biome> IS_AQUATIC_ICY = register("is_aquatic_icy");
+	/**
+	 * Biomes that exist on the shoreline of a body of water.
+	 */
+	public static final TagKey<Biome> IS_BEACH = register("is_beach");
+	/**
+	 * Biomes densely populated with deciduous trees.
+	 */
+	public static final TagKey<Biome> IS_FOREST = register("is_forest");
+	public static final TagKey<Biome> IS_BIRCH_FOREST = register("is_birch_forest");
+	public static final TagKey<Biome> IS_OCEAN = register("is_ocean");
+	public static final TagKey<Biome> IS_DESERT = register("is_desert");
+	public static final TagKey<Biome> IS_RIVER = register("is_river");
+	public static final TagKey<Biome> IS_SWAMP = register("is_swamp");
+	public static final TagKey<Biome> IS_MUSHROOM = register("is_mushroom");
+	public static final TagKey<Biome> IS_UNDERGROUND = register("is_underground");
+	public static final TagKey<Biome> IS_MOUNTAIN = register("is_mountain");
+
+	public static final TagKey<Biome> IS_HOT = register("is_hot");
+	public static final TagKey<Biome> OVERWORLD_IS_HOT = register("is_hot/overworld");
+
+	public static final TagKey<Biome> IS_TEMPERATE = register("is_temperate");
+	public static final TagKey<Biome> OVERWORLD_IS_TEMPERATE = register("is_temperate/overworld");
+
+	public static final TagKey<Biome> IS_COLD = register("is_cold");
+	public static final TagKey<Biome> OVERWORLD_IS_COLD = register("is_cold/overworld");
+
+	public static final TagKey<Biome> IS_WET = register("is_wet");
+	public static final TagKey<Biome> OVERWORLD_IS_WET = register("is_wet/overworld");
+
+	public static final TagKey<Biome> IS_DRY = register("is_dry");
+	public static final TagKey<Biome> OVERWORLD_IS_DRY = register("is_dry/overworld");
+
+	public static final TagKey<Biome> IS_VEGETATION_SPARSE = register("is_sparse_vegetation");
+	public static final TagKey<Biome> OVERWORLD_IS_VEGETATION_SPARSE = register("is_sparse_vegetation/overworld");
+
+	public static final TagKey<Biome> IS_VEGETATION_DENSE = register("is_dense_vegetation");
+	public static final TagKey<Biome> OVERWORLD_IS_VEGETATION_DENSE = register("is_dense_vegetation/overworld");
+
+	public static final TagKey<Biome> CONIFEROUS_IS_TREE = register("is_tree/coniferous");
+	public static final TagKey<Biome> SAVANNA_IS_TREE = register("is_tree/savanna");
+	public static final TagKey<Biome> JUNGLE_IS_TREE = register("is_tree/jungle");
+	public static final TagKey<Biome> DECIDUOUS_IS_TREE = register("is_tree/deciduous");
```

Comment:
I'm really not a fan of the `IS_` prefixes in general, but if we're using them in the field names they should be in the natural order for English.

These should be `ARE_TREES_*` or something akin to that.

Reply:
Yeah so that's what BiomeTags in Yarn is doing right now. It sticks the folder name as a suffix and it's super ugly
![image](https://github.com/FabricMC/fabric/assets/40846040/1a1dfc26-ca91-460d-9355-3742a8a5cd4c)

I am matching that cause an earlier request asked me to match the yarn names. Unless we make an exception for these biome tags and sandstone tags to be in a more readable format. 

I did raise an issue report to Yarn repo about these biome tags and if they change the order, I'll definitely for sure change  this PR's biome tags to match
https://github.com/FabricMC/yarn/issues/3661

Reply:
In this case I think the yarn names will be changed as they aren't following the current naming conventions, just a matter of someone taking the time to do it.

Reply:
Since it seems like the yarn names will change (hopefully before 1.21), I adjusted the problematic biome tag field names to be in a more readable format. Hopefully this helps out!